client/stylesheets/_toCat.scss

```diff
@@ -4813,13 +4814,14 @@ On click the images will be displayed at normal size to complete the effect
 /*Options DropDown*/
 
 .optionsClick {
-	margin:100px auto;
-	width:200px;
+    width: 100%;
+    margin-top: 23px;
+    margin-bottom: 200px;
 }
 
 .optionsClickv ul {
 	position:relative;
-	font-weight:900;
+	font-weight:400;
```

Comment:
Line should be indented with spaces, not tabs<br>Colon after property should be followed by one space